tests/topotests/lib/bgp.py

```diff
@@ -989,6 +989,14 @@ def __create_bgp_unicast_address_family(
             if "no_allowas_in" in peer:
                 allow_as_in = peer["no_allowas_in"]
                 config_data.append("no {} allowas-in {}".format(neigh_cxt, allow_as_in))
+
+            if "shutdown" in peer:
+                shut_val = peer["shutdown"]
+                if shut_val is True:
+                    config_data.append("{} shutdown".format(neigh_cxt))
+                elif shut_val is False:
+                    config_data.append("no {} shutdown".format(neigh_cxt))
```

Comment:
```py
if "shutdown" in peer:
  config_data.append("{} {} shutdown".format("no" if peer["shutdown"] else "", neigh_cxt))
```
Don't compare against true and false in conditions

Reply:
Thanks @qlyoung , updated code. Please check.